docs/markdown/snippets/build-target-found.md

```diff
@@ -0,0 +1,16 @@
+## New `build target` methods
+
+The [`build target` object](Reference-manual.md#build-target-object) now supports
+the following two functions, to ensure feature compatebility with
+[`external program` objects](Reference-manual.html#external-program-object):
+
+- `found()`: Always returns `true`. This function is meant
+  to make executables objects feature compatible with
+  `external program` objects. This simplifies
+  use-cases where an executable is used instead of an external program.
+
+- `path()`: **(deprecated)** does the exact same as `full_path()`.
+  **NOTE:** This function is solely kept for compatebility
+  with `external program` objects. It will be
+  removed once the, also deprecated, corresponding `path()` function is
```

Comment:
So `path` will be removed when  `path` is removed? That seems a bit weird. What does this actually refer to?

Reply:
It refers to the already deprecated `path()` method in `external program` object. The idea here is that the `path` method in `build target` objects is only required because `external program` objects also have a `path()` method.

A new user should never actually use this new `path` method intentionally.

Reply:
Yes, but the text should say "corresponding `path()` function in `external_program` is removed".

Reply:
Yeah, updated.